### ISSUE TITLE
Update Publications, News, and People pages

### ISSUE THREAD

**Issue by kdahlquist** [kdahlquist/GRNmap#342]
Reminder to update the News page with items about the SoCal Systems biology conference and the LMU URS.  Update the publication page with presentations/posters.  Update the People page with the new picture of the GRNmap team (coding+data analysis) from the symposium (#301).

**Comment by kdahlquist** [comment]
@johnllopez616 can get started on this one by adding himself to the People pages of both GRNmap and GRNsight.  Eddie A. and Eddie B. need to be moved to the Former student category.

**Comment by johnllopez616** [comment]
Added myself to people pages and removed former students. I had a difficult time installing Jekyll on my computer so next time I'll make changes using the computers in either the Dahlquist Lab or Keck Lab. Also I could not locate the image for the group.

**Comment by kdahlquist** [comment]
I don't think that you need Jekyll actually installed on your computer.  GitHub will do that part automatically.   @NAnguiano made a Jekyll tutorial on the GRNsight wiki here:  https://github.com/dondi/GRNsight/wiki/Jekyll-Tutorial

I will need to give you some guidance as to where to find the content to upload to the site. I haven't had a chance to do that this week, though.

**Comment by kdahlquist** [comment]
https://davidburela.wordpress.com/2015/11/28/easily-install-jekyll-on-windows-with-3-command-prompt-entries-and-chocolatey/

@johnllopez616 will follow these instructions to install Jekyll on the computers in the Dahlquist Lab, LSB 263

**Comment by johnllopez616** [comment]
The documentation page has been updated and as with #239, development will be underway when I set up Jekyll and a proper dev environment using a Mac in the Keck Lab. I will keep you posted as to when that happens.

**Comment by johnllopez616** [comment]
Jekyll has been installed on a mac in the Keck lab and expect development to continue today on this and #239

**Comment by kdahlquist** [comment]
* News page needs to be updated with release 1.8
* About page needs to be updated with release 1.8
* Downloads page needs to be updated with release 1.8
* News and publication pages need to be updated with presentations at conferences, in 2017, @kdahlquist will make list
* Links page needs a link to the Biomath modeling course from Spring 2017: https://openwetware.org/wiki/BIOL398-05/S17

**Comment by kdahlquist** [comment]
Here is the list of conference presentations in 2017.  You will need to change the format to be consistent with what is already there:

Southern California Conference for Undergraduate Research, California State Polytechnic University, Pomona, November 18, 2017, Chukwuemeka E Azinge*, Justin Kyle T. Torres*, John David N. Dionisio, Ben G. Fitzpatrick, Kam D Dahlquist, Restructuring the Data Architecture of GRNmap, a Gene Regulatory Network Modeling Application (poster)

Ninth Annual Undergraduate Research Symposium, Loyola Marymount University, March 25, 2017

Kristen M. Horstmann*, Ben G. Fitzpatrick, and Kam D. Dahlquist, Systems modeling and statistical analysis allows comparison in the response to cold shock (talk)
in Saccharomyces cerevisiae between Hap4 and randomly generated networks

Brandon J. Klein*, Natalie E. Williams*, Ben G. Fitzpatrick, and Kam D. Dahlquist, Dynamical systems modeling of six related small gene regulatory networks suggest that the transcription factors Cin5, Gln3, Hmo1, and Yhp1 play a role in controlling the cold shock response in Saccharomyces cerevisiae (poster)

Margaret J. O’Neil*, Ben G. Fitzpatrick, Kam D. Dahlquist, Using Graph Statistics to Investigate the Properties of Six Candidate Gene Regulatory Networks for Controlling the Cold Shock Response in Saccharomyces cerevisiae (poster)

Trixie Anne M. Roque*, Chukwuemeka E. Azinge*, Justin Kyle T. Torres*, John David N. Dionisio, Ben G. Fitzpatrick, Kam D. Dahlquist, Restructuring the Data Architecture of GRNmap, a Gene Regulatory Network Modeling Application (poster)

Natalie E. Williams*, Ben G. Fitzpatrick, Kam D. Dahlquist, Comparison of the regulatory dynamics of related small gene regulatory networks that control the cold shock response in Saccharomyces cerevisiae (talk)

7th Annual Southern California Systems Biology Conference, University of California, Irvine, January 28, 2017

Kristen M. Horstmann*, Margaret J. O’Neil*, Ben G. Fitzpatrick, Kam D. Dahlquist, Dynamical systems modeling and gene regulatory network structure analysis reveals Hap4's role in regulating the response to cold shock in Saccharomyces cerevisiae (poster)

Natalie E. Williams*, Brandon J. Klein*, Ben G. Fitzpatrick, and Kam D. Dahlquist, Dynamical systems modeling of six related small gene regulatory networks suggest that the transcription factors Cin5, Hmo1, Msn2, and Yhp1 play a role in controlling the cold shock response in Saccharomyces cerevisiae, (poster)

Some of the abstracts are on the wiki--you should be able to match them with the names/titles.  Posters and talk slides should be found on the master branch documents directory (either under poster or talks).  You may not be able to find all of it, but post what you can.  I'll help you track down whatever is missing after you get it up.

**Comment by kdahlquist** [comment]
On People page, can you move Kristen Horstmann and Natalie Williams to the former students category (ending point for research should be Spring 2017)

Please add to current students:
Lauren M. Kelly (LMU Biology '19) Spring 2017 - Present

**Comment by kdahlquist** [comment]
I've uploaded all the pictures I could find to the assets > img directory.  I think that's it for pictures for now.  We'll take more at the LMU symposium in March.

**Comment by johnllopez616** [comment]
Updates have been made as above, changes have been committed, and website is ready for review!

**Comment by kdahlquist** [comment]
I will have to get to this later.  Probably during Spring Break.
